addons/lunch/data/mail_template_data.xml

```diff
@@ -4,10 +4,10 @@
     <record id="lunch_order_mail_supplier" model="mail.template">
         <field name="name">Lunch: Supplier Order</field>
         <field name="model_id" ref="lunch.model_lunch_supplier"/>
-        <field name="email_from">{{ ctx['order']['email_from'] }}</field>
-        <field name="partner_to">{{ ctx['order']['supplier_id'] }}</field>
-        <field name="subject">Orders for {{ ctx['order']['company_name'] }}</field>
-        <field name="lang">{{ ctx.get('default_lang') }}</field>
+        <field name="email_from">{{ ctx.get('order') and ctx['order']['email_from'] }}</field>
+        <field name="partner_to">{{ ctx.get('order') and  ctx['order']['supplier_id'] }}</field>
+        <field name="subject">Orders for {{ ctx.get('order') and  ctx['order']['company_name'] }}</field>
```

Comment:
This is only ever called with a value for 'order' in the context. It makes sense for it to be assumed non-null, as a user if you want to use it elsewhere you can edit it yourself to be able to.

This change doesn't really fix it properly because now email_from is just None and instead of a traceback, I get a weird behavior where my email is not sent.